lib/grizzly/command_handler.ex

```diff
@@ -15,3 +15,30 @@ defmodule Grizzly.CommandHandler do
 
   @optional_callbacks handle_command: 2
 end
+
+# defmodule Grizzly.CommandHandler do
+#   @moduledoc """
+#   Behaviour for how commands should handle incoming Z-Wave messages
+#   """
+
+#   alias Grizzly.ZWave.Command
+
+#   @doc """
+#   """
+#   @callback init(any()) :: {:ok, state :: any()}
+
+#   @doc """
+#   Called when a Z/IP Packet with the ACK Response flag is received from
+#   Z/IP Gateway. This will only be called when the sequence number in the ACK Response
+#   packet matches the sequence number from the original command packet.
+#   """
+#   @callback handle_ack(state) :: {:complete, :ok} | {:continue, true, state} | {:continue, state}
+#             when state: var
+
+#   @doc """
+#   Called when a Z/IP Packet containing a Z-Wave command is received from
+#   Z/IP Gateway.
+#   """
+#   @callback handle_command(Command.t(), state) :: {:complete, Command.t()} | {:continue, state}
+#             when state: var
+# end
```

Comment:
What is all this commented code? Is it needed?

Reply:
Oops, forgot to delete it. 😅